src/index.html

```diff
@@ -1,13 +1,325 @@
+                The app senses when you're nearby to unlock automatically. GPS tracking so you know where your bike is and can track it anytime.
+              </p>
+            </div>
+            <div class="grid grid--tablet grid--desktop">
+              <img
+                class="page__section-detail page__section-detail--size-1 grid__item--tablet-1-4 grid__item--desktop-1-8"
+                src="./images/image-3.png"
+                alt="detail photo 1"
+              >
+              <img
+                class="page__section-detail page__section-detail--size-2 grid__item--tablet-5-6 grid__item--desktop-9-12"
+                src="./images/image-4.png"
+                alt="detail photo 2"
+              >
+              <h3 class="page__section-subtitle grid__item--tablet-1-3 grid__item--desktop-1-4">
+                Range & Integrated lights
+              </h3>
+              <p class="page__section-text grid__item--tablet-1-6 grid__item--desktop-1-8">
+                The removable battery has up to 70km battery autonomy and weighs only 2.4 kg. Lights integrated into the frame give you always-on visibility day and night.
+              </p>
+            </div>
+            <div class="grid grid--tablet grid--desktop">
+              <img
+                class="page__section-detail page__section-detail--size-2 grid__item--tablet-1-2 grid__item--desktop-1-4"
+                src="./images/image-5.png"
+                alt="detail photo 1"
+              >
+              <img
+                class="page__section-detail page__section-detail--size-1 grid__item--tablet-3-6 grid__item--desktop-5-12"
+                src="./images/image-6.png"
+                alt="detail photo 2"
+              >
+              <h3 class="page__section-subtitle grid__item--tablet-1-5 grid__item--desktop-1-6">
+                Hydraulic disc brakes & Lightweight
+              </h3>
+              <p class="page__section-text page__section-text--padding grid__item--tablet-1-6 grid__item--desktop-1-8">
+                Brakes with total stopping power the second you make contact. The removable battery has up to 70km battery autonomy and weighs only 2.4 kg. Lights integrated into the frame give you always-on visibility day and night.
+              </p>
+            </div>
+
+            <div class="grid grid--tablet grid--desktop">
+              <a href="#" class="button grid__item--tablet-2-5 grid__item--desktop-5-8" >
+                Explore
+              </a>
+            </div>
+          </div>
+        </div>
+      </section>
+
+      <section class="page__section" id="contacts">
+        <div class="container">
+          <h2 class="page__section-title page__section-title--padding">
+            Contact us
+          </h2>
+
+
+          <div class="grid grid--tablet grid--desktop">
+            <div class="grid__item--tablet-1-3 grid__item--desktop-1-5">
+              <form action="#" method="get">
```

Comment:
I don't know, how i can do this:
Page shouldn't be reloaded on form submit (https://developer.mozilla.org/en-US/docs/Web/API/Event/preventDefault)